src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2366,6 +2367,9 @@ protected virtual void MarkReflectionLikeDependencies (MethodBody body)
 			if (HasManuallyTrackedDependency (body))
 				return;
 
+			var scanner = new ReflectionMethodBodyScanner (this);
```

Comment:
Eventually, this should only run when we detect stores into unsafe locations (unsafe method callsite/unsafe method return value/unsafe field) as part of the method body marking.